environment.yml

```diff
@@ -20,7 +20,7 @@ dependencies:
   - rapidnj
   - h5py
   - requests
-  - pp-sketchlib >=2.0.0
+  - pp-sketchlib >=2.0.1
```

Comment:
I think also needs to be set in __init__.py (and while you're there, bump to v2.6.0!)

Reply:
Set in 3b562b9